src/Platforms/MySQLPlatform.php

```diff
@@ -195,7 +195,8 @@ public function getListTableForeignKeysSQL($table, $database = null)
 
         return $sql . ' AND k.table_schema = ' . $databaseNameSql
             . ' /*!50116 AND c.constraint_schema = ' . $databaseNameSql . ' */'
-            . ' AND k.`REFERENCED_COLUMN_NAME` is not NULL';
+            . ' AND k.`REFERENCED_COLUMN_NAME` is not NULL'
+            . ' ORDER BY k.`ORDINAL_POSITION` ASC';
```

Comment:
`ASC` is redundant.

Reply:
@morozov I just checked other similar statements in MySQLPlatform and all of them add `ASC` at the end, see: https://github.com/doctrine/dbal/blob/3.1.3/src/Platforms/MySQLPlatform.php#L364
https://github.com/doctrine/dbal/blob/3.1.3/src/Platforms/MySQLPlatform.php#L157

personally, I also prefer to add the direction and not count on the default direction staying ASC, though it is extremely unlikely that MySQL will change that 😄 

However, as mentioned for other similar statements it is with ASC, so I would prefer to keep it, so it is in sync with the others.

Reply:
It is still redundant. The fact that this practice is used in other places doesn't justify using it here. Please remove it.

Reply:
I have now removed it.

Though, personally I think staying in line with existing code syntax used (in this case SQL for MySQL platform) should be more important here, than removing redundancy but I am fine with removing it. Also I wouldn't say it is redundant, it is specific. Redundant would be if the same ORDER BY would be added twice, after all there is the extermely unlikely scenario, that MySQL adds an option to change the default order by direction and somebody decides to use it but as mentioned extremely unlikely :smile: